### PR TITLE
Document models in readme

### PR DESCRIPTION
**Pull request by olethanh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +25,10 @@ unit that is part of a canonical hierarchy. Each one of these org. units can hav
 territory. The mobile data collection tool can be used to enrich this hierarchy with additional GPS
 coordinates, names corrections, etc ... which can then be validated by officials of the organizations
 in question through the web dashboard. This leads to continuous improvements of the geographic references
-available through the routine activities already planned (e.g. locating and registering health facilities while
-investigating malaria cases).
+available through the routine activities already planned. e.g. locating and registering health facilities while
+investigating malaria cases.
 
-The tool has been used in multiple data collection efforts, notably in the domain of Performance Based Financing
+The tool has been used in multiple data collection efforts. Notably in the domain of Performance Based Financing
```
Comment (madewulf) [inline README.md:31]: This is not a phrase if you do that, there is no verb :-)

File: README.md [hunk 2]
```diff
@@ -51,12 +51,52 @@ via an API implemented via Django rest framework, all data is stored in Postgres
...
+---------------------
+
+Some terminology in Iaso come from DHIS2, some from ODK which mean that it can be a bit confusing.
+We will highlight some equivalences that might help you.
+
+This is not (yet) the complete Data Model, but here are the main concepts/model in Iaso:
+
+* Iaso is multi tenant. Tenant are called and represented by the model `Account`. It represents roughly one client org or country. It also represent the natural limit of right for a user.
+* Each Django's User has a linked Iaso `Profile` that link it to an `Account` and store extra parameters for the user.
+* Each tenant can have multiple `Project`. Projects are linked to one android version App via the `app_id`. To control what a user can see from that app.
```
Comment (madewulf) [inline README.md:71]: not a phrase either :-)

File: README.md [hunk 3]
```diff
@@ -51,12 +51,52 @@ via an API implemented via Django rest framework, all data is stored in Postgres
...
+* `DHIS2` is a standard database and web UI in the industry to handle Health Data. Iaso can import and export data to it.
```
Comment (madewulf) [inline README.md:72]: It's not a database, it's a java app built on top of Postgres

File: README.md [hunk 4]
```diff
@@ -51,12 +51,52 @@ via an API implemented via Django rest framework, all data is stored in Postgres
...
+* `OrgUnit` (DHIS2 terminology) is a Node of the GeoRegistry tree. e.g a particular Country, City or Hospital. each belonging to each other via a `parent` relationship.
```
Comment (madewulf) [inline README.md:73]: Actually, oranizational unit is a term largely used in a lot of fields

File: README.md [hunk 5]
```diff
@@ -51,12 +51,52 @@ via an API implemented via Django rest framework, all data is stored in Postgres
...
+   * They can have a type `OrgUnitType` e.g. Country, City, Hospital
+   *  and can belong to multiple `Group`, e.g. Urban Region or Campaign 2017
+   * DHIS2 has the concept of `Group` but not `Type` so when importing from a DHIS2 Instance all the type will be Unknown and OrgUnit will belong to group like `Clinic`
+   * `GroupSet` are Group of group. Used when we export Group to DHIS2
+  * OrgUnit may have a position in space, it can be an area, the  `geom` field is then used, or just a Point, the `location` field is then used.
+    * It's technically possible to have both
+    * a OrgUnit may have no Geographical info
+    * a OrgUnit may be Geographically outside its parent.
```
Comment (madewulf) [inline README.md:81]: why the upper case to Geo*

File: README.md [hunk 6]
```diff
@@ -51,12 +51,52 @@ via an API implemented via Django rest framework, all data is stored in Postgres
...
+* `DataSource` links OrgUnit and Group imported from the same source, e.g a DHIS2 instance, a CSV or a GeoPackage.
+   * A `source_ref` on the imported instance is used to keep the reference from the original source, so we can match it again in the future (when updating the import or exporting it back)
+   * `SourceVersion` is used to keep each version separated. e.g each time we import from DHIS2 we create a new version.
+   * OrgUnit (for their parent) and Group should only reference other OrgUnit and Group in the same version. (This is not enforced everywhere yet)
+* `Task` are asynchronous function that will be run by a background worker in production. eg: Importing Data from DHIS2. see Worker section below for more info.
+* `Form` is the definition of a Form. Which question should be asked and how they are presented. The model contain the meta data, the actual definition is done in a `XSLForm` as an attached file.
```
Comment (madewulf) [inline README.md:87]: Not a phrase

**Comment by olethanh** [comment]
corrected the remarks and added section on  database dump and restore

**Pull request merged**